sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/QueryPlanRetriever.java

```diff
@@ -23,10 +23,13 @@
     private static final String SUPPORTED_QUERY_FEATURES = QueryFeature.Aggregate.name() + ", " +
                                                                QueryFeature.CompositeAggregate.name() + ", " +
                                                                QueryFeature.MultipleOrderBy.name() + ", " +
+                                                               QueryFeature.MultipleAggregates.name() + ", " +
                                                                QueryFeature.OrderBy.name() + ", " +
                                                                QueryFeature.OffsetAndLimit.name() + ", " +
                                                                QueryFeature.Distinct.name() + ", " +
-                                                               QueryFeature.Top.name();
+                                                               QueryFeature.GroupBy.name() + ", " +
+                                                               QueryFeature.Top.name() + ", " +
+                                                               QueryFeature.NonValueAggregate.name();
```

Comment:
We need to add tests for this if you are going to turn the flag on.

Reply:
Yeah, added tests for NonValue and Multi Aggregates in AggregateQueryTests